config/manager/manager.yaml

```diff
@@ -1,75 +1,52 @@
 apiVersion: v1
 kind: Namespace
 metadata:
+  annotations:
+    workload.openshift.io/allowed: "management"
   labels:
-    control-plane: controller-manager
-    app.kubernetes.io/name: ksan
-    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/name: ksan-storage-operator
+    security.openshift.io/scc.podSecurityLabelSync: "false"
+    pod-security.kubernetes.io/enforce: "privileged"
+    pod-security.kubernetes.io/warn: "privileged"
+    pod-security.kubernetes.io/audit: "privileged"
+    openshift.io/cluster-monitoring: "true"
   name: system
 ---
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: controller-manager
+  name: operator
   namespace: system
   labels:
-    control-plane: controller-manager
-    app.kubernetes.io/name: ksan
-    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/name: ksan-storage-operator
 spec:
   selector:
     matchLabels:
-      control-plane: controller-manager
+      app.kubernetes.io/name: ksan-storage-operator
   replicas: 1
   template:
     metadata:
       annotations:
-        kubectl.kubernetes.io/default-container: manager
+        target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
       labels:
-        control-plane: controller-manager
+        app.kubernetes.io/name: ksan-storage-operator
+        app.kubernetes.io/part-of: ksan-storage
+        app.kubernetes.io/component: ksan-storage-operator
     spec:
-      # TODO(user): Uncomment the following code to configure the nodeAffinity expression
```

Comment:
Hey! an example of how you should handle architectures :D
I suspect we want to set boundaries for amd64 at the minimum.

Reply:
afaik kubesan supports arm64 and amd64 currently. I don't know the long term plan for architectures though. If we plan to support all 4, we don't need this block. If we only plan to support arm and amd, then it makes sense to add it imho

Reply:
@brandisher do you know what the arch boundaries we plan to support are?

Reply:
we need to support amd64 first and on inital release (Dev Preview).
arm64 is second prio and could be added post-ga. Also fine to add it right away if we can get it kind of for free with little effort or delays.
currently no long term plans for ppc64 or s390, but this might change based on customer demand.